relay-server/src/endpoints/mod.rs

```diff
@@ -53,6 +54,12 @@ where
     let store_routes = Router::new()
         // Legacy store path that is missing the project parameter.
         .route("/api/store/", store::route(config))
+        // cron level routes.  These are user facing APIs and as such avoid the project ID and
+        // they support optional trailing slashes.
+        .route("/api/cron/:monitor_slug/:sentry_key", cron::route(config))
+        .route("/api/cron/:monitor_slug/:sentry_key/", cron::route(config))
+        .route("/api/cron/:monitor_slug", cron::route(config))
+        .route("/api/cron/:monitor_slug/", cron::route(config))
```

Comment:
Why do we have to support trailing `/` and no trailing `/` URLs?  Have the routes without trailing slashes been  already exposed to customers and customer got used to using them by now? 

If not,  IMO we have to keep our endpoints consistent, and steer the customers to use only one defined route. And currently we have `/` at the end, which I would also lean to.

Reply:
I generally think, from a devex perspective, it's confusing to get errors for missing (or extra) slashes at the end of an URL. Although, I would have imagined we handle this in a cleaner way that adding both URL variants.

Reply:
Agree would be nice to handle this in a more general way